js/classes/enterRoom.js

```diff
@@ -59,10 +56,19 @@ class CurrentRoom {
     this._connectedRooms = generatePaths();
     this._contents = initContents(this._room.contentTypes);
 
-    setTimeout(() => {
-      this._poison = generateEffect(poison, this._player);
-      this._player.effects.push(this._poison);
-    }, 10000);
+    console.log(this._contents);
+
+    // TODO: remove this before pushing to prod
+    window.Room = this;
+
+    // setTimeout(() => {
+    //   this._poison = generateEffect(poison, this._player);
+    //   this._player.effects.push(this._poison);
+    // }, 10000);
+  }
```

Comment:
when you comment a block out, just toss a comment above it explaining why. No need to change this now, but for future reference